ci/base/base.yml

```diff
@@ -61,7 +61,7 @@ builds:
     with_dot_net: #@ True
     #@yaml/map-key-override
     params:
-      CMAKE_CONFIGURE_FLAGS: "-A x64 -Tv141,version=14.16,host=x64 -DCMAKE_SYSTEM_VERSION=10.0.16299.0"
```

Comment:
I thought host=x64 was still necessary for VS2017 builds?

Reply:
I don't think we need this anymore. I'm able to build fine using VS 2017 without it. I've verified that my local build was using the 32 bit linker, and everything still builds and all tests pass. When I do a dumpbin /headers on the 32 bit linker, I see:
     "Application can handle large (>2GB) addresses"
Maybe this has been fixed.

Reply:
Good catch though. If this really is no longer needed we need to remove the check for it, which is a fatal error.

Reply:
Okay, so some Googling tells me that for cmake 3.13 or newer, we need to ditch "Win64" in the -G parameter and use "-A x64" to configure 64-bit builds:  https://stackoverflow.com/questions/28350214/how-to-build-x86-and-or-x64-on-windows-from-command-line-with-cmake
And "-T host=x64" is used only to specifically force 64-bit toolset, so I don't suppose this is anything other than maybe a small performance hit.